tests/spec/fs.writeFile-readFile.spec.js

```diff
@@ -99,3 +99,25 @@ describe('fs.writeFile, fs.readFile', function() {
     });
   });
 });
+
+describe('fsPromises.Write', function() {
+  it('should write, read a utf8 file without specifying utf8 in writeFile', function() {
+    //var fs = util.fs();
+    //var contents = 'This is a file.';
```

Comment:
You don't want to leave commented code in a PR like this.  If you don't need the code, remove it.  There's no need to keep it around, since we can go back in time with git to recover old code if necessary.